base/sysinfo.jl

```diff
@@ -196,9 +196,10 @@ function _cpu_summary(io::IO, cpu::AbstractVector{CPUinfo}, i, j)
 end
 
 """
-    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = cpu_info())
+    Sys.cpu_summary(io::IO=stdout, cpu::AbstractVector{CPUinfo} = Sys.cpu_info())
```

Comment:
Perhaps we should remove the whitespaces in the second keyword argument? It took me quite a few seconds to parse that line and to see whether there is an assignment going on here...

Reply:
Good idea.